templates/default/jaas_client_config.erb

```diff
@@ -0,0 +1,23 @@
+// Auto-generated by Chef, do not edit!
+
+// JAAS configuration for Kafka client
+KafkaClient {
+    com.sun.security.auth.module.Krb5LoginModule required
+<% node["kafka"]["kerberos"]["krb5_properties"].each do |key, value| -%>
+    <%= key %>=<%= (value.to_s == 'true' || value.to_s == 'false') ? value : "\"#{value}\"" %>
```

Comment:
Yes if the value is either true or false (String or literal boolean) we don't wrap it in quotes. For all other values we do